runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/environment/DockerEnvironmentFactory.java

```diff
@@ -225,6 +233,11 @@ private static ServerFactory getServerFactory() {
 
   /** Provider for DockerEnvironmentFactory. */
   public static class Provider implements EnvironmentFactory.Provider {
+    private final boolean cleanup;
```

Comment:
I suspect there will be many other docker options that we eventually would want to expose, especially for local runs. Mounts, networking, etc. Perhaps it would make sense to use a structure similar to ManualDockerEnvironmentOptions mentioned below to avoid updating the plumbing multiple times.

Reply:
Makes sense. Will update the file name.